pages/docs/concepts/asyncapi-document/server-security.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Server Security 
+weight: 200
+---
+
+Server security refers to the measures and practices implemented to protect servers from unauthorized access, data breaches, and other security threats. Server security involves implementing various security mechanisms to ensure the confidentiality, integrity, and availability of server resources.
+
+In the context of AsyncAPI, securing servers ensures secure exchange of messages between clients and servers. While also protecting sensitive data, preventing unauthorized access, and maintaining the overall security of the API or server.
+
+You can secure a server by using the `security` property where you define or declare which security schemes can be used with the server in context. Each `server` in the AsyncAPI document can have one or more security schemes declared. A security scheme defines a security requirement that must be satisfied to authorize an operation such as a API key or username and password. 
+
+Here is a example of adding security to your server:
+```yml
+asyncapi: '3.0.0'
```

Comment:
it is definitely not enough to just change version to 3.0 and have a valid document

- there is no longer `url` on server but `host`
- security is defined in different way now, look at 3.0 example https://github.com/asyncapi/spec/blob/next-major-spec/examples/streetlights-kafka.yml

please validate your documentation with https://studio.asyncapi.com/ before submitting for review

Reply:
Made the required changes to the example using the reference provided. Thankyou :)